doc/src/template.md

```diff
@@ -1 +1,66 @@
-# Challenge Template
+# Challenge XXXX[^challenge_id]: Challenge Title
+
+- **Status:** *One of the following: [Open | Resolved | Expired]*
+  - **Solution:** *Option field to point to the PR that solved this challenge.*
+- **Tracking Issue:** *Link to issue*
+- **Start date:** *YY/MM/DD*
+- **End date:** *YY/MM/DD*
+
+-------------------
+
+
+Status: [Open | Resolved | Expired]:
+Tracking issue:
+Start date:
+Due date:
+Author(s):
```

Comment:
This somehow overlaps with the lines above?!

Reply:
My bad. :)